src/exercises/exerciseTypes/multipleChoiceAudio/MultipleChoiceAudio.js

```diff
@@ -0,0 +1,273 @@
+import { useState, useEffect } from "react";
```

Comment:
how come this is a new file? where was it before? if it's a rename, why don't we see the other file deleted? or renamed? this is a bit puzzling.

Reply:
When i renamed the files appeared in git that the old files had been deleted and the current files had been added..don't know why this is the case but they contain the same code